demos/ReactorChat/ReactorChat-Server/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>ReactorChat</artifactId>
+        <groupId>com.salesforce.servicelibs</groupId>
+        <version>0.7.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>ReactorChat-Server</artifactId>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <!-- Import dependency management from Spring Boot -->
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>1.4.2.RELEASE</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.salesforce.servicelibs</groupId>
+            <artifactId>grpc-spring</artifactId>
+            <version>${grpc.contrib.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-autoconfigure</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-logging</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>4.2.0.RELEASE</version>
+        </dependency>
+        <!-- Force Reactor 3 instead of Reactor 2 which is pulled by the current Spring Boot-->
+        <dependency>
```

Comment:
Or maybe use Spring Boot 2.0.0.M6 ?
@rmichela WDYT ?

Reply:
I'm fine with either. Reactor 3 is the most exciting part. Is Reactor 3 backward compatible with Reactor 2?

Reply:
> Is Reactor 3 backward compatible with Reactor 2?

I don't think so : Flux used to be Stream, Mono used to be Promise, etc...
So the code generated by the plugin only works with Reactor 3. Maybe that's worth mentioning.

Reply:
Note that Reactor 2 is quite old and legacy (last release in April 2016)